phpstan-baseline.neon

```diff
@@ -26,7 +26,7 @@ parameters:
 			path: admin/admin-base.php
 
 		-
-			message: "#^Parameter \\#6 \\$callback of function add_submenu_page expects callable\\(\\)\\: mixed, array\\{\\$this\\(PLL_Admin_Base\\), 'languages_page'\\} given\\.$#"
+			message: "#^Parameter \\#6 \\$callback of function add_submenu_page expects ''\\|\\(callable\\(\\)\\: mixed\\), array\\{\\$this\\(PLL_Admin_Base\\), 'languages_page'\\} given\\.$#"
```

Comment:
Couldn't we fix the error instead by truly creating the missing method even if it does nothing? I have abused the system in the past, knowing the method cannot be called from `PLL_Admin` but only from `PLL_Settings`.

Reply:
Done in [25115f6](https://github.com/polylang/polylang/pull/1469/commits/25115f6f9299ba39abc6b0f64149a68d07669b01).